testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeHudi.java

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
I would remove the commit body "Add Hudi product tests" since the commit title is self descriptive.

Reply:
Makes sense. I will edit it after squashing all commits once the PR is near landing.